charts/adguard-home/values.yaml

```diff
@@ -1,3 +1,6 @@
+# upgrade strategy type (e.g. Recreate or RollingUpdate)
+strategyType: RollingUpdate
```

Comment:
Why don't we go ahead and change the default to Recreate?  You bring up a good point about a workload using persistent data with RollingUodate not being a good idea.

Reply:
Sure I could change the default to Recreate. I would say, we should leave it adjustable via `values.yaml`. Maybe someone wants to experiment with that to have an "HA"-Setup.